docs/configuring-socorro.rst

```diff
@@ -27,13 +27,37 @@ The Socorro systemd service scripts use envconsul
 (https://github.com/hashicorp/envconsul) to read the configuration from Consul
 and set the environment.
 
-Below is the minimum viable configuration to get collection and
-processing working on a single node via Consul's REST interface::
+To keep track of your configuration, it's recommended to make a "socorro-config"
+directory and store these values there, and then load those into Consul as
+a separate step.
 
-    curl -s -X PUT -d "socorro.webapi.servers.WSGIServer" localhost:8500/v1/kv/socorro/collector/web_server.wsgi_server_class
-    curl -s -X PUT -d "/path/to/your/symbols" localhost:8500/v1/kv/socorro/processor/processor.raw_to_processed_transform.BreakpadStackwalkerRule.processor_symbols_pathname_list
+Below is the minimum viable configuration to get collection working on a
+single node:
 
-Note that Consul also has a Web UI you can use to get/set keys if you prefer.
+.. code-block:: bash
+
+    # Run collector in WSGI mode, instead of the default dev server
+    web_server__wsgi_server_class='socorro.webapi.servers.WSGIServer'
+
+Put this into a file named "collector.conf" in your socorro-config folder. 
+
+For processing to work, the path to your debug symbols must be provided:
+
+.. code-block:: bash
+
+    # This should be replaced with the path to your debug symbols.
+    processor__raw_to_processed_transform__BreakpadStackwalkerRule__processor_symbols_pathname_list='@@@PATH_TO_YOUR_SYMBOLS@@@'
```

Comment:
the ```reference_value_from``` can be used to create a shorter form of the configuration key.  However, I do not think that it would make a new value that was short enough.  ```reference_value_from``` specifies an alternative namespace (or nested namespaces) where a default for the option can be found.  For example:
``` python
n0.n1.n2.add_option('o1', default=99, reference_value_from='r')
```
produces: 
```python
#n0.n1.n2.o1=99
r.o=99
```
on the command line, the value can be changed by referencing it using either path:
```bash
$ ./my_app.py --n0.n1.n2.o1=65
$ ./my_app.py --r.o1=65
# the original form overrides the reference_value_from. the value will be 65 in the next invocation
$ ./my_app.py --n0.n1.n2.o1=65 --r.o1=222  
```
reference values can be shared between several options.  This is what we do with database and other crash storage credentials.  That allows multiple sections of code that need access to PG, to declare their need within their own namespace context, but all pick up their config from a common location.  

The reference value system doesn't allow for changing the name of a option.  ```reference_value_from``` always is a list of namespaces into which the original name of the option will be inserted.  

It would not bee too difficult to create a new ```alias=``` parameter for the Option object to create a true shortcut to a long deeply nested value.